systemtest/src/test/java/io/strimzi/systemtest/connect/ConnectBuilderST.java

```diff
@@ -0,0 +1,328 @@
+
+        KafkaConnectResource.create(KafkaConnectResource.kafkaConnect(clusterName, 1)
+            .editMetadata()
+                .addToAnnotations(Annotations.STRIMZI_IO_USE_CONNECTOR_RESOURCES, "true")
+            .endMetadata()
+            .editOrNewSpec()
+                .addToConfig("key.converter.schemas.enable", false)
+                .addToConfig("value.converter.schemas.enable", false)
+                .addToConfig("key.converter", "org.apache.kafka.connect.storage.StringConverter")
+                .addToConfig("value.converter", "org.apache.kafka.connect.storage.StringConverter")
+                .withNewBuild()
+                    .withPlugins(PLUGIN_WITH_TAR_AND_JAR)
+                    .withNewDockerOutput()
+                        .withNewImage(imageName)
+                    .endDockerOutput()
+                .endBuild()
+                .withNewInlineLogging()
+                    .addToLoggers("connect.root.logger.level", "INFO")
+                .endInlineLogging()
+            .endSpec()
+            .build(), true);
+
+        Map<String, Object> connectorConfig = new HashMap<>();
+        connectorConfig.put("topics", topicName);
+        connectorConfig.put("level", "INFO");
+
+        LOGGER.info("Creating EchoSink connector");
+        KafkaConnectorResource.create(KafkaConnectorResource.kafkaConnector(echoConnector, clusterName)
+            .editOrNewSpec()
+                .withClassName(ECHO_SINK_CLASS_NAME)
+                .withConfig(connectorConfig)
+            .endSpec()
+            .build());
+
+        String deploymentName = KafkaConnectResources.deploymentName(clusterName);
+        Map<String, String> connectSnapshot = DeploymentUtils.depSnapshot(deploymentName);
+
+        LOGGER.info("Checking that KafkaConnect API contains EchoSink connector and not Camel-Telegram Connector class name");
+        String plugins = cmdKubeClient().execInPod(kafkaClientsPodName, "curl", "-X", "GET", "http://" + KafkaConnectResources.serviceName(clusterName) + ":8083/connector-plugins").out();
+
+        assertFalse(plugins.contains(CAMEL_TELEGRAM_CLASS_NAME));
+        assertTrue(plugins.contains(ECHO_SINK_CLASS_NAME));
+
+        LOGGER.info("Adding one more connector to the KafkaConnect");
+        KafkaConnectResource.replaceKafkaConnectResource(clusterName, kafkaConnect -> {
+            kafkaConnect.getSpec().getBuild().getPlugins().add(secondPlugin);
+        });
+
+        DeploymentUtils.waitTillDepHasRolled(deploymentName, 1, connectSnapshot);
+
+        LOGGER.info("Creating Camel-Telegram connector");
+        KafkaConnectorResource.create(KafkaConnectorResource.kafkaConnector(camelConnector, clusterName)
+            .editOrNewSpec()
+                .withClassName(CAMEL_TELEGRAM_CLASS_NAME)
+                .withConfig(connectorConfig)
+            .endSpec()
+            .build());
```

Comment:
Do we really have to create this? It will not work anyway since you do not pass it credentials to Telegram bot. If you wanna create it, maybe you can use some connector which does not really end up in error.

Reply:
Yes, good point, I just wanted to check that when other plugin is added, we can create the connector. So maybe I'll try another one

Reply:
MAybe you can try https://camel.apache.org/camel-kafka-connector/latest/connectors/camel-http-kafka-sink-connector.html ... if it doesn't get any message, it should not call any HTTP endpoint, so it does not matter what you configure it with. 

Ot the Camel Timer Source connector ... you can configure it to send out periodically messages with timestamp:

```yaml
apiVersion: kafka.strimzi.io/v1alpha1
kind: KafkaConnector
metadata:
  name: camel-timer-connector
  labels:
    strimzi.io/cluster: my-connect
spec:
  class: CamelTimerSourceConnector
  tasksMax: 1
  config:
    topics: timer-topic
    camel.source.path.timerName: timer
    key.converter: org.apache.kafka.connect.storage.StringConverter
    value.converter: org.apache.kafka.connect.json.JsonConverter
    value.converter.schemas.enable: false
    transforms: HoistField,InsertField,ReplaceField
    transforms.HoistField.type: org.apache.kafka.connect.transforms.HoistField$Value
    transforms.HoistField.field: originalValue
    transforms.InsertField.type: org.apache.kafka.connect.transforms.InsertField$Value
    transforms.InsertField.timestamp.field: timestamp
    transforms.InsertField.static.field: message
    transforms.InsertField.static.value: "Hello World"
    transforms.ReplaceField.type: org.apache.kafka.connect.transforms.ReplaceField$Value
    transforms.ReplaceField.blacklist: originalValue
```